docs/whirlpool/docs/03-Getting Started/01-Environment Setup.md

```diff
@@ -0,0 +1,135 @@
+# Environment setup
+
+This document covers the essential setup required to start building on Orca’s SDK using the Whirlpools protocol. It includes wallet setup, RPC client configuration, airdropping tokens for testing, and the basics of interacting with the Solana ecosystem.
+
+## Prerequisites
+
+Before you start, ensure you have Node.js installed on your machine. Download it from the official website: https://nodejs.org/.
+
+## 1. Initialize a new project
+Create a new project directory:
+
+```bash
+mkdir whirlpool-env-setup
+cd whirlpool-env-setup
+```
+
+Initialize a new Node.js project:
+
+```bash
+npm init -y
+```
+
+Install the necessary packages:
+
+```bash
+npm install typescript ts-node @orca-so/whirlpools-client @solana/web3.js fs
+```
+
+Initialize the project as a TypeScript project
+
+```bash
+npx tsc --init
+```
+
+## 3. Wallet Creation and Management
+
+To simplify wallet management, we’ll generate a wallet programmatically and store its private key in a .env file for future use.
+
+First, create a `wallet.ts` file in your current directory and add the following code.
+
+```tsx title="wallet.ts"
+import { Keypair } from '@solana/web3.js';
+import * as fs from 'fs';
+import * as dotenv from 'dotenv';
+import * as path from 'path';
+
+dotenv.config();
+
+const ENV_FILE_PATH = path.resolve(__dirname, './.env');
+
+function createWallet(): Keypair {
+  const keypair = Keypair.generate();
+  const secretKey = `[${keypair.secretKey.toString()}]`;
+  fs.appendFileSync(ENV_FILE_PATH, `\nPRIVATE_KEY=${secretKey}\n`);
+  console.log(`New wallet created with address: ${keypair.publicKey.toBase5()}`);
+  return keypair;
+}
+
+export function getWallet(): Keypair {
+  const privateKeyString = process.env.PRIVATE_KEY;
```

Comment:
I would try to keep this as simple as possible. Do we really need to store the key in the env? I would just do `generateKeypair`, check the balance and do an airdrop

Reply:
I will update this and all related comments.